client/src/project/new/components/Visibility.tsx

```diff
@@ -256,49 +274,51 @@ const EditVisibility = ({
       ""
     );
 
-  if (!isLoadingProject && !isFetchingProject && !projectData)
-    return (
-      <WarnAlert dismissible={false}>
-        Knowledge Graph integration must be enabled to change visibility from
-        the RenkuLab web interface.
-      </WarnAlert>
-    );
+  const isLoadingSomething =
+    isLoadingNamespace || isLoadingProject || isLoadingIndexingStatus;
 
-  if (isLoadingProject && isFetchingProject && !projectData) return <Loader />;
+  const content = isLoadingSomething ? (
+    <>
+      <label>Visibility</label>
```

Comment:
DOM semantics (#2653):

```suggestion
      <div className="form-label">Visibility</div>
```